bridgeifns.py

```diff
@@ -0,0 +1,268 @@
+
+    # Get load config from config_file
+    if not Path(config_file).exists():
+        return False, messages[3011]
+    with Path(config_file).open('r') as file:
+        config = json.load(file)
+
+    # Get the ipv6_subnet from config_file
+    ipv6_subnet = config.get('ipv6_subnet', None)
+    if ipv6_subnet is None:
+        return False, messages[3012]
+    # Verify the ipv6_subnet value
+    try:
+        ipaddress.ip_network(ipv6_subnet)
+    except ValueError:
+        return False, messages[3013]
+
+    # Get the PodNet Mgmt ips from ipv6_subnet
+    podnet_a = f'{ipv6_subnet.split("/")[0]}10:0:2'
+    podnet_b = f'{ipv6_subnet.split("/")[0]}10:0:3'
+
+    # Get `podnet_a_enabled` and `podnet_b_enabled`
+    podnet_a_enabled = config.get('podnet_a_enabled', None)
+    if podnet_a_enabled is None:
+        return False, messages[3014]
+    podnet_b_enabled = config.get('podnet_b_enabled', None)
+    if podnet_a_enabled is None:
+        return False, messages[3015]
+
+    # First run on enabled PodNet
+    if podnet_a_enabled is True and podnet_b_enabled is False:
+        enabled = podnet_a
+        disabled = podnet_b
+    elif podnet_a_enabled is False and podnet_b_enabled is True:
+        enabled = podnet_b
+        disabled = podnet_a
+    elif podnet_a_enabled is True and podnet_b_enabled is True:
+        return False, messages[3016]
+    elif podnet_a_enabled is False and podnet_b_enabled is False:
+        return False, messages[3017]
+    else:
+        return False, messages[3018]
+
+    # Define payload
+
+    payload_bridge_exists = f'ip link show {bridgename}'
+    payload_namespace_exists = f'ip link show {bridgename}'
+    payload_1 = f'ip link add {bridgename}.{namespace} type veth peer name {namespace}.{bridgename}'
+    payload_2 = f'ip link set dev {bridgename}.{namespace} master {bridgename}'
+    payload_3 = f'ip link set dev {namespace}.{bridgename} netns {namespace}'
+    payload_4 = f'ip netns exec {namespace} ip link set dev {namespace}.{bridgename} up'
+
+    #Check if bridgename and namespaces already exists or not
+    try:
+        #Check if bridgename exists if true it will return 0
+        exit_code_bridge, _, _ = comms_ssh(
+            host_ip='localhost',
+            payload=payload_bridge_exists,
+            username='robot',)
+
+        #Check if namespace exists if true it will return 0
+        exit_code_namespace, _, _ = comms_ssh(
+            host_ip='localhost',
+            payload=payload_namespace_exists,
+            username='robot',)
+
+        # If there are names then the exit code would be 0 and what we want is both bridge and namespace to give exit code 1
+        # which means neither exists.
+        if exit_code_namespace and exit_code_bridge:
+            already_exists = False
+        else:
+            already_exists = True
+
+    except CouldNotConnectException:
+        already_exists = True
+
+    if already_exists:
+        return False, messages[3000]
```

Comment:
using message `3000` for every failure is not the right way, we want to know where it failed exactly so use the specific message please. Also why you are not following directorymain primitive? Do follow it please.

Reply:
If message 3000 is not used then remove it from messages dict.